README.md

```diff
@@ -48,8 +48,8 @@ repo key and setup a new sources.list entry:
 
 ```shell
 wget -qO- https://repos.influxdata.com/influxdb.key | sudo tee /etc/apt/trusted.gpg.d/influxdb.asc >/dev/null
-source /etc/os-release
-echo "deb https://repos.influxdata.com/${ID} ${VERSION_CODENAME} stable" | sudo tee /etc/apt/sources.list.d/influxdb.list
+export VERSION_ID=$(lsb_release -si); export VERSION_CODENAME=$(lsb_release -sc)
```

Comment:
the value of ${ID} from `/etc/os-release` is lower case already

Reply:
true, but trying to source the whole /etc/os-release` file was causing problems (getting an empty value). Maybe `ID` is too generic?

Reply:
Could you share your /etc/os-release? I'm curious what was erroring out.

In any case, I am trying to update our repos to make them more generic to not depend on any codename, id, etc. so we can avoid this in the first place.

Reply:
ok